tests/integration/run.sh

```diff
@@ -89,22 +89,22 @@ export MAIN_SERVER_CONFIG_DIR
 export REAL_FEDERATED_SERVER_CONFIG_DIR
 
 export NEXTCLOUD_CONFIG_DIR="$MAIN_SERVER_CONFIG_DIR"
-OVERWRITE_CLI_URL=$(${ROOT_DIR}/occ config:system:get overwrite.cli.url)
+MAIN_OVERWRITE_CLI_URL=$(${ROOT_DIR}/occ config:system:get overwrite.cli.url)
+MAIN_SKELETON_DIR=$(${ROOT_DIR}/occ config:system:get skeletondirectory)
 ${ROOT_DIR}/occ config:system:set overwrite.cli.url --value "http://localhost:8080/"
+if [[ "$MAIN_SKELETON_DIR" != "" && -d $MAIN_SKELETON_DIR ]]; then
```

Comment:
Should it be simply tested if the value is set, rather than if it is set and the directory exist? If it is set but it does not exist the tests would use it rather than the default skeleton directory*, which would make the tests fail if they are expecting some file from the default skeleton directory.

*Unless there is a fallback in Nextcloud code in case the configured skeleton directory does not exist, I have not checked that.

Reply:
yeah that might have been an attempt to find the root cause on my side. The problem turned out to be the overwriting from 2 different servers :see_no_evil: